src/UserBundle/EventListener/UserRegistrationListener.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace UserBundle\EventListener;
+
+use FOS\UserBundle\FOSUserEvents;
+use FOS\UserBundle\Event\FormEvent;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use UserBundle\Entity\User;
+
+
+class UserRegistrationListener implements EventSubscriberInterface
+{
+    /**
+     * @var ContainerInterface $container
+     */
+    protected $container;
+
+    public function __construct(Container $container)
+    {
+        $this->container = $container;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return array(
+            FOSUserEvents::REGISTRATION_SUCCESS => [
+                ['onRegistrationSuccess', -10],
+            ],
+        );
+    }
+
+    public function onRegistrationSuccess(FormEvent $event)
+    {
+
+        /** @var $user User */
+        $user = $event->getForm()->getData();
+        // Here we take the civility value to return a boolean
+        $gender = $user->getCivility();
+        if ($gender == 'm'){
+            $user->setCivility(true);
+        } else if ($gender == 'f'){
+            $user->setCivility(false);
+        }
+        // Here we take a string to set it Uppercase the first character
```

Comment:
little mislanguage here, you should write : 
"Here we capitalize the first letter of a string"